src/System.IdentityModel.Tokens/AsymmetricSignatureProvider.cs

```diff
@@ -238,11 +255,15 @@ public override byte[] Sign(byte[] input)
             {
                 throw new ObjectDisposedException(GetType().ToString());
             }
-
+#if DNXCORE50
+            if (rsaCng != null)
+                return rsaCng.SignData(input, hash, RSASignaturePadding.Pkcs1);
```

Comment:
@brentschmaltz CoreFX's RSA is aligned with .net 4.6.

`RSASignaturePadding.Pkcs1` is the only mode that RSACryptoServiceProvider understands, which is why it previously wasn't an option.  RSACng supports mode kinds of padding, so it's now a parameter on Sign and Verify.